document/CoreFormat.mdk

```diff
@@ -218,7 +218,9 @@ Suppose a tool is invoked with the command line
     C:\Tools\CodeScanner\CodeScanner.exe /input C:\Code\*.cc
 ```
 
-Then the value of the `commandLineArguments` property should be `"/input C:\Code\*.cc"`.
+Then the value of the `commandLineArguments` property should be `"C:\Tools\CodeScanner\CodeScanner.exe /input C:\Code\*.cc"`. In 
+circumstances where the fully-qualified path to the tool .exe name provides undesirable information disclosure, this information
```

Comment:
".exe" is Windows-specific. Also I don't think fully-qualified is hyphenated. Perhaps: "In circumstances where the fully qualified path to the tool provides..."